app/scripts/modules/core/insight/insight.module.ts

```diff
@@ -0,0 +1,14 @@
+import * as angular from 'angular';
```

Comment:
we've been just `import {module} from 'angular'` instead of the whole thing, then `module('spinnaker.core.insight', [` below